edb/lib/ext/auth.edgeql

```diff
@@ -105,10 +98,73 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
             create annotation std::description :=
                 "ID for client provided by auth provider";
         };
+
+        create required property display_name: std::str {
+            set readonly := true;
+            set protected := true;
+        };
+
+        create property additional_scope: std::str;
+    };
+
+    # create type ext::auth::OpenIdProviderConfig
```

Comment:
We're not shipping this? cc @scotttrinh

Reply:
This is arbitrarily defined OIDC providers, not the three we have built-in (apple, google, azure).

Reply:
James, let's just kill these lines then. I don't like shipping commented out snippets in code.